Documentation/PageTsconfig/Mod/WebList.rst

```diff
@@ -200,6 +200,62 @@ Example: Hide the column selector
 
     mod.web_list.displayColumnSelector = 0
 
+..  _pageTsConfigWebList-downloadPresets:
+
+downloadPresets
+===============
+
+..  confval:: downloadPresets.[table]
+    :name: mod-web-list-downloadPresets
+    :type: array of presets
+
+    ..  versionadded:: 13.2
+        A new feature has been introduced, adding the ability to pick the exported
+        data columns from a list of configurable presets.
+
+    This property adds presets of preselected fields to the download area in
+    the :guilabel:`Web > List` backend module.
+
+    Those presets can be configured via page TSconfig, and can also be
+    overridden via user TSconfig (for example, to expand certain presets
+    only to specific users).
+
+    Each entry of :typoscript:`mod.web_list.downloadPresets`
+    defines the table name on the first level, followed by
+    any number of presets.
+
+    Each preset contains a :typoscript:`label` (the displayed name of the preset,
+    which can be a locallang key), a comma-separated list of each column that
+    should be included in the export as :typoscript:`columns` and optionally
+    a :typoscript:`identifier`. In case :typoscript:`identifier` is not provided,
+    the identifier is generated as hash of the :typoscript:`label` and
+    :typoscript:`columns`.
+
+    Since any table can be configured for a preset, any extension
+    can deliver a defined set of presets through the
+    :file:`EXT:my_extension/Configuration/page.tsconfig` file and
+    their table name(s).
+
+    ..  todo:: Link event after it is merged
+
+    Additionally, the list of presets can be manipulated via the PSR-14 event
+    :php:`BeforeRecordDownloadPresetsAreDisplayedEvent`.
+
+..  _pageTsConfigWebList-downloadPresets-example:
+
+Example: Create download presets for table page
+-----------------------------------------------
+
+..  literalinclude:: _WebList/_downloadPresets.tsconfig
+    :caption: EXT:my_extension/Configuration/page.tsconfig
+
+This can be manipulated with user TSConfig by adding the :typoscript:`page.`
```

Comment:
Thank you for sharing the link to our glossary ;-) 

```suggestion
This can be manipulated with user TSconfig by adding the :typoscript:`page.`
```

Reply:
Copy and pasted this from the changelog entry. You want to fix it there?